### DISCUSSION TITLE
All this mess! My mess ofcourse 🥴

### DISCUSSION THREAD

**Discussion by Dawiducha** [community/community#144229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi guys im need help whit this all mess! Im so confuse i got lot of tokens to take and im just dont now how to do! Please team im realy need help! Im can pay 15% from my al assets! Greathings 
Dawid Jadkowski

**Comment by shinybrightstar** [comment]
Hi @Dawiducha,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Dawiducha** [comment]
Hi yes thanks guys for contact,my live is a bit up side down i got memory
problems and honestly im so confuse whit all this...im sorry but im want
ask YOU guys for help to take my assets and make everything right,for help
im can give 10%for community like a donation or something for as community
sorry for my english 🤗 im hopeful and Thank you guys!
Greathings
Dawid Jadkowski

czw., 21 lis 2024, 14:55 użytkownik >
napisał:
